src/core.sh

```diff
@@ -70,24 +70,29 @@ fi
 search_libc() {
   if [[ -z ${FS_libc} ]]; then
     # if a specific search path is given, use it
-    LIBC_SEARCH_PATH=/
     if [[ -n "${LIBC_FILE}" ]]; then
       if [[ -f "${LIBC_FILE}" ]]; then
         FS_libc=${LIBC_FILE}
       elif [[ -d "${LIBC_FILE}" ]]; then
         LIBC_SEARCH_PATH=${LIBC_FILE}
       fi
+    # otherwise use ldd to get the libc location
+    elif [[ -f $(ldd "$(command -v grep)" 2> /dev/null | grep 'libc\.so' | cut -d' ' -f3) ]]; then
```

Comment:
Can you add `ldd` to the command list on L56?

Reply:
Done :)